app/use_cases/get_enrolments.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import BaseModel
+
+from app.domain.entities.enrolment_request import EnrolmentFilters
+from app.repositories.enrolment_repo import EnrolmentRepo
+from app.responses import ResponseFailure, ResponseSuccess, SuccessType
+
+
+class GetEnrolments(BaseModel):
```

Comment:
GetEnrolmentRequests better

Reply:
I think we shouldn't have `request` in a usecase name. But I'm changing it to `GetEnrolmentAuthorisations`